build/release/Makefile

```diff
@@ -61,7 +61,8 @@ FLAVORS ?= output
 override BRANCH_NAME := pr/$(BRANCH_NAME)
 endif
 
-DOCKER_REGISTRY ?= rook
+# DOCKER_REGISTRY ?= rook
```

Comment:
All changes in this makefile will be reverted before merge, right? Looks like they are all for temporary testing.